src/libraries/Common/src/System/Net/ArrayBuffer.cs

```diff
@@ -57,11 +57,13 @@ public void Dispose()
 
         public int ActiveLength => _availableStart - _activeStart;
         public Span<byte> ActiveSpan => new Span<byte>(_bytes, _activeStart, _availableStart - _activeStart);
-        public ReadOnlySpan<byte> ActiveReadOnlySpan => new ReadOnlySpan<byte>(_bytes, _activeStart, _availableStart - _activeStart);
```

Comment:
I believe @wfurt added `ActiveReadOnlySpan` recently for `SslStream`. Is it no longer used there?

Reply:
It is used in SslStream. That is why the builds are failing.

Reply:
Ah, I'll revert.  It was showing up as dead code in System.Net.Http; didn't realize it was used elsewhere.